### PR TITLE
Bill Trustworthy's Discount Firearms and Enterprising Detritus

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
You know the drill by now, trash vendor in all but name, but now I know how to branch

## Why It's Good For The Game
more gat access

## Changelog
:cl:
add: Added Bill Trustworthy's Discount Firearms and Enterprising Detritus, a gun vendor, now with hacking enabled
del: Removed Gladstone shotgun from FS vendor and moved it to Bill Trustworthy's
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
the gladstone makes more sense in FS. krammer should be moved to bills

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1485,6 +1484,52 @@
...
+					//obj/item/ammo_magazine/cspistol = 12,
+					/obj/item/ammo_magazine/srifle = 12,
+					/obj/item/ammo_magazine/smg = 12,
+					/obj/item/part/armor = 20, 
+					/obj/item/part/gun = 20,
+					/obj/item/weapon/gun/projectile/automatic/ak47/fs/brush = 4,
+					/obj/item/weapon/gun/energy/retro = 4,
+					/obj/item/weapon/gun/projectile/mk58  = 2,
+					/obj/item/weapon/gun/projectile/mk58/wood = 2,
+					//obj/item/weapon/gun/projectile/mandella = 4,
```
[the remark this answers is not in the document]
Comment (zombiemimic) [reply code/game/machinery/vending.dm:1505]: Yeah, it is, but it might be confusing.

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1485,6 +1484,50 @@
 	idle_power_usage = 211
 	vendor_department = DEPARTMENT_CIVILIAN
 
+/obj/machinery/vending/Billomat
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:1487]:
```suggestion
/obj/machinery/vending/billomat
```

**Review by Firefox13 — APPROVED** [review]

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1485,6 +1484,50 @@
 	idle_power_usage = 211
 	vendor_department = DEPARTMENT_CIVILIAN
 
+/obj/machinery/vending/billomat
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:1487]:
```suggestion
/obj/machinery/vending/Billomat
```
ok it decided to break itself

**Pull request merged**